api/src/test/java/org/motechproject/nms/api/osgi/MobileAcademyControllerBundleIT.java

```diff
@@ -79,12 +85,27 @@ public void testSetValidBookmark() throws IOException, InterruptedException {
     }
 
     @Test
-    public void testGetCourse() throws IOException, InterruptedException {
+    public void testGetCourseNotPresent() throws IOException, InterruptedException {
 
         String endpoint = String.format("http://localhost:%d/api/mobileacademy/course",
                 TestContext.getJettyPort());
+        HttpGet request = RequestBuilder.createGetRequest(endpoint);
+
+        assertTrue(SimpleHttpClient.execHttpRequest(request, HttpStatus.SC_INTERNAL_SERVER_ERROR, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD));
```

Comment:
`<nitpick level="11">`
set the vertical margin on your editor to 113 and it'll make reviewing the commits easier (caus' you know, no horizontal scrolling necessary)
`</nitpick>`